playwright.config.ts

```diff
@@ -50,6 +50,7 @@ const config: PlaywrightTestConfig = {
     command: 'npm run preview',
     port: 3000,
   },
+  workers: 1,
```

Comment:
Any reason to set this back to 1? We were successfully running tests before locally with multiple works. Historically running with multiple workers was a bit of a kludgey fix to get around some issues with the test infrastructure and tests themselves that were not understood but we took some time to debug those and multiple workers _should_ work now.

Reply:
@duranb told me to add it when I had trouble developing my test and I was trying to debug the problem. I accidentally pushed this file when I committed the requested changes. I will revert it.

Reply:
Got it, thanks!